### DISCUSSION TITLE
Analysing tools

### DISCUSSION THREAD

**Discussion by Coocys** [ppy/osu#15413] [Ideas] [answered]
Hey,

I just want to share a ideas that would solve some issues that bother me about osu! sice I started to play.

In my opinion the replay system should provide some kind of overview of the "mistakes/errors" you made and sum it up in the timeline i some extend (maby you cant configure yourself what you want to see)
Maby extend the hit graph thats new in lazer by showing the misses too.
This would solve the endless scrubbing of replays to find what went wrong.

I find it hard sometimes to pinpoint what I am doing wrong to not hit the circle, thats why I am sugesting this.

kind regards

**Comment by peppy** [comment]
Definitely something we will be looking to add. Everything has been made to support this.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/5681

**Reply by Coocys** [reply]
Oh I searched if someone made something like this but dont found it.
Ty for commenting it.
Sorry for the duplicate ^^
